src/app/malaria-101/activity-3/activity-3.component.ts

```diff
@@ -32,9 +37,14 @@ export class OddOneOutComponent implements OnInit {
     public completed = false;
     public alerts: any;
     public solutions = '';
+    public level: number;
+    public levelConfig = [4, 6, 8]; // no of questions depending upon the level
```

Comment:
Number of questions is okay, but don't we have separate questions for each level ?

Reply:
Yup. Added.